Source/src/components/ComponentCamera.h

```diff
@@ -64,6 +64,8 @@ namespace Hachiko
         bool preview_cam = false;
         float3 camera_pinned_pos = float3::zero;
 
+        bool isMoving = false;
```

Comment:
this should be called i_moving. Also if it only applies to the editor camera i would store this variable in ModuleCamera